_docs/interfaces.md

```diff
@@ -38,8 +34,8 @@ The TCPIP client interface connects to a TCPIP socket to send commands and recei
 | Read Port | Port to read telemetry from (can be the same as write port) | Yes |
 | Write Timeout | Number of seconds to wait before aborting the write. Pass 'nil' to block on write. | Yes |
 | Read Timeout | Number of seconds to wait before aborting the read. Pass 'nil' to block on read. | Yes |
-| Stream Protocol Type | See Streams and Stream Protocols. | Yes |
-| Stream Protocol Arguments | See Streams and Stream Protocols for the arguments each stream protocol takes. | Yes |
+| Stream Protocol Type | See Streams and Stream Protocols. | No |
```

Comment:
Still referring to stream protocol here and a few other places.   Otherwise looks good.